ui/app/pages/confirm-transaction/confirm-transaction.component.js

```diff
@@ -70,7 +65,8 @@ export default class ConfirmTransaction extends Component {
     if (isTokenMethodAction) {
       getTokenParams(to)
     }
-    this.props.setTransactionToConfirm(transactionId || paramsTransactionId)
+    const txId = transactionId || paramsTransactionId
+    if (txId) this.props.setTransactionToConfirm(txId)
```

Comment:
Why is this needed?

Reply:
It gets rid of the following error otherwise thrown on occasion by that call:
`[2019-08-21 20:32:23.690][e2e] chrome-extension://aohcjlklkjjpdgebibknknkinbdpdmoe/ui.js 0:265094 "Transaction with id undefined not found"`

Reply:
I thought that was a problem with my changes but that error is currently thrown by the `7.1.0` RC and I think by current prod: https://circleci.com/gh/MetaMask/metamask-extension/102563?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Can we split this out into its own PR?

Reply:
Done: https://github.com/MetaMask/metamask-extension/pull/7053